packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -83,76 +84,43 @@ export async function exportSurveyResponses(req, res) {
   const workbook = { SheetNames: [], Sheets: {} };
 
   try {
+    const variablesExtractor = new SurveyResponseVariablesExtractor(models);
     let country;
     let entities;
-    let reportName;
-    if (viewId) {
-      const dashboardReport = await models.dashboardReport.findById(viewId);
-      reportName = dashboardReport.viewJson.name;
-    }
+    const reportName = viewId && (await models.dashboardReport.findById(viewId).viewJson.name);
```

Comment:
```suggestion
    const reportName = viewId && (await models.dashboardReport.findById(viewId)).viewJson.name;
```
What about this with the bracket moved? I think you were trying to get `viewJson` of a promise.

Reply:
Nice! Works now!